Add an rpm package building system

corollari opened the pull request:
While tinkering with your package to get it working on Fedora, I decided to build an rpm package for it, which should make it easier for other Fedora users to install (among other things, this package sets the terminal command used by PyLNP, as for some reason PyLNP is incapable of detecting the terminal on Fedora and guessing which command should be used for it turned out to be really bothersome, speaking from experience here :laughing: ). I've also added some instructions on README explaining how the package should be built.

Additionally, please consider changing the list of dependencies listed for Fedora in the wiki to replace `tkinter` with `python3-tkinter`, as `tkinter` doesn't exist as a package. If possible I'd do so myself but I don't have write access to the wiki.

I've also added a minor edit on the commands present in the readme.

McArcady commented on .gitignore:
```diff
@@ -14,4 +14,5 @@ config/configure.in
 .config.*
 *.bz2
-
+lazy-newb-pack-*.src.rpm
```
We could call the package "linux-dwarf-pack-*", since it goes by this name on the bay12 forum and on DFFD.

McArcady commented:
Great! Nice job setting this up.
I propose we call the resulting package 'LinuxDwarfPack' or 'linux-dwarf-pack' since it goes by that name on DFFD and the Bay12 forums (see my comment in the commits)
The dependencies in the wiki have been updated.

corollari commented:
Cool, just changed it to `linux-dwarf-pack` (went with this one instead of the camel case one in order to follow the conventions specified in [Fedora's packaging guidelines](https://fedoraproject.org/wiki/Packaging:Naming?rd=Packaging:NamingGuidelines#Separators)).

I've also gone ahead and split the dependencies better into build-time and usage-time groups, following #17.
